kubernetes/samples/charts/traefik/samples/host-routing.yaml

```diff
@@ -1,35 +1,34 @@
-apiVersion: networking.k8s.io/v1beta1
-kind: Ingress
+# Copyright (c) 2018, 2020, Oracle Corporation and/or its affiliates.
+# Licensed under the Universal Permissive License v 1.0 as shown at https://oss.oracle.com/licenses/upl.
+
+apiVersion: traefik.containo.us/v1alpha1
+kind: IngressRoute
```

Comment:
Does this also work for older versions of Traefik? Or are we dropping support for older versions of Traefik?

Reply:
Note that the MII sample uses older version of Traefik ingresses - it directly supplies the YAML in `kubernetes/samples/scripts/create-weblogic-domain/model-in-image/ingresses`.

Reply:
We are not supporting older 1.x version of Traefik  as decided by PM

Reply:
Then the kubernetes/samples/scripts/create-weblogic-domain/model-in-image/ingresses need to be updated.  This is a three step process: alter the script that generates the ingresses, rerun the generator, check in a new version of the ingresses to the same locations.   Also, you'll need to update the MII sample documentation that describes the ingresses.

Reply:
I will send you the specific steps in slack.

Reply:
I will create another PR to update the MII sample.

Reply:
Seems OK, as long as the MII sample and ItMiiSample continue to work with this PR in place.  But if the sample or ItMiiSample break, then IT and samples break. They have a dependency on the old 'Quick Start' instructions for starting Traefik.

Reply:
Updated the MiiSample code and related doc files.